spec/contexts/movies/commands/update_spec.rb

```diff
@@ -0,0 +1,45 @@
+require 'rails_helper'
+require 'faker'
+
+RSpec.describe Contexts::Movies::Commands::Update do
+  describe '#call' do
+    subject(:command) do
+      described_class.new
+    end
+
+    before { create(:movie) }
+
+    let(:first_movie) { Movie.first }
+    let(:params) do
+      {
+        id: first_movie.id,
+        imdb_id: first_movie.imdb_id,
+        title: first_movie.title,
+        rating: first_movie.rating,
+        rank: first_movie.rank,
+        year: first_movie.year,
+        data: nil
+      }
+    end
+
+    context 'when params valid' do
+      it 'updates the record' do
+        old_year = params[:year]
+        params[:year] = 1998
+        command.call(params)
+        new_year = Movie.first.year
+        expect(new_year).not_to eq(old_year)
```

Comment:
and then check that updated is same as param
```suggestion
        expect(updated_movie.year).to eq(params[:year])
```